src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/BlobRestConnector.java

```diff
@@ -43,15 +40,17 @@
 
   @Autowired
   CloseableHttpClient httpClient;
-  
+
   public BlobApplicationAware get(BlobApplicationAware blob) {
 
     String uri = baseUrl + "/" + blobBasePath + "/" + blob.getContainer() + "/" + blob.getBlob();
     final HttpGet getBlob = new HttpGet(uri);
     getBlob.setHeader(new BasicHeader("Ocp-Apim-Subscription-Key", blobApiKey));
     
     try {
-      httpClient.execute(getBlob, new FileDownloadResponseHandler(new FileOutputStream(Path.of(blob.getTargetDir(), blob.getBlob()).toFile())));
+      OutputStream result = httpClient.execute(getBlob,
```

Comment:
@TommasoLencioni I suspect a problem in branching since these lines of code are older than this PR